x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/components/custom_urls/description.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiDescribedFormGroup, EuiFormRow, EuiLink } from '@elastic/eui';
+import { metadata } from 'ui/metadata';
+
+const docsUrl = `https://www.elastic.co/guide/en/elastic-stack-overview/${metadata.branch}/ml-configuring-url.html`;
+
+export const Description: FC = memo(({ children }) => {
+  const title = i18n.translate(
+    'xpack.ml.newJob.wizard.jobDetailsStep.additionalSection.customUrls.title',
+    {
+      defaultMessage: 'Custom URLs',
+    }
+  );
+  return (
+    <EuiDescribedFormGroup
+      fullWidth
+      idAria="description"
+      title={<h3>{title}</h3>}
+      description={
+        <FormattedMessage
+          id="xpack.ml.newJob.wizard.jobDetailsStep.additionalSection.customUrlsSelection.description"
+          defaultMessage="Custom URLs provide links from the anomalies table in the Anomaly Explorer or Single Metric Viewer window to Kibana dashboards, the Discovery page, or external websites. {learnMoreLink}"
```

Comment:
Suggest cutting this down to `Provide links from anomalies to Kibana dashboards, the Discovery page, or other web pages.`, with the Learn more kept as-is.

Reply:
Fixed in https://github.com/elastic/kibana/pull/51281/commits/8438634bf5c7f0afd22b6c9f71756514bad0beda